tools/docs/build_docs.py

```diff
@@ -46,9 +46,14 @@
 from tensorflow_docs.api_generator import generate_lib
 from tensorflow_docs.api_generator import parser
 from tensorflow_docs.api_generator import public_api
+from tensorflow_docs.api_generator import utils
 
 from tensorflow.python.util import tf_inspect
 
+
+# tfio doesn't eagerly import submodules.
+utils.recursive_import(tfio)
```

Comment:
This doesn't actually work yet (Crashes with NotImplementedErrors), I've got a fix inflight to the tensorflow_docs repo that will fix this.